Changelog.md

```diff
@@ -21,6 +21,7 @@
 - Fix bug where gzipped binary feedback files were not unzipped correctly (#6283)
 - Fix bug where remark request summary table wasn't being rendered correctly (#6284)
 - Fix bug where test results were being associated with the wrong test runs (#6287)
+- Filter out invalid properties on assignment upload (#6224)
```

Comment:
Changelog entries should really describe what the change means for the user of MarkUs (whether that is an end-user, admin, or developer). This describes what has been changed but doesn't really describe how that changes things for a user.

In the case of bugfixes, I like to describe the bug that was fixed in the changelog as opposed to the mechanism used to fix it. Similarly for features, I like to describe the feature that was added instead of how it was done in the code. 

In this case, can we change this to something like "Fix bug where assignments could not be uploaded if unexpected properties were present in the uploaded file content". That's a bit wordy maybe, but something like that